packages/flutter/lib/src/widgets/visibility.dart

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
```

Comment:
The overall class comment below still mentions that this may use the Opacity widget behind the scenes. Maybe update that? We should probably still link to the Opacity widget in the see also section, though.

Reply:
Same fro the class comment on SliverVisibility

Reply:
I do see links to the Opacity/SliverOpacity widgets, but I can't see any comments that directly state that this uses opacity

Reply:
I interpreted "These widgets provide some of the facets of this one:" as such. But I don't feel strongly about this comment.

Reply:
I think I still want to link to those widgets, I don't see that as necessarily claiming the implementation uses Opacity but providing a hint on a similar way to achieve the same effect.